transport/http2_server.go

```diff
@@ -277,7 +277,7 @@ func (t *http2Server) operateHeaders(frame *http2.MetaHeadersFrame, handle func(
 		}
 		s.ctx, err = t.inTapHandle(s.ctx, info)
 		if err != nil {
-			// TODO: Log the real error.
+			grpclog.Printf("transport: http2Server.operateHeaders failed to called inTapHandle: %v", err)
```

Comment:
"failed to called" doesn't seem right. Perhaps, something like Error from InTapHandle ?